README.md

```diff
@@ -7,7 +7,10 @@ This repository contains an Ingress controller for Kubernetes that works by depl
 
 ## Prerequisites
 
-Tested with Kubernetes clusters running version 1.8 and later.
+- Tested with Kubernetes clusters running version `1.8` through `1.10`.
+- The Ingress Controller supports the following versions of Kong:
+   - Kong Community Edition(CE) `0.13.x`
+   - Kong Enterprise Edition(EE) >= `0.32.x`
```

Comment:
I'd suggest being specific here regarding Kong EE versions - this currently implies that _all_ versions of Kong EE >= 0.32 will be compatible, and... well, that may or may not be the case.